packages/grub2/config/grub.cfg

```diff
@@ -74,12 +57,8 @@ menuentry "${display_name} (fallback)" --id fallback {
 }
 
 menuentry "${display_name} recovery" --id recovery {
-  # label and recoverylabel are kept around for backward compatibility
-  if [ -n "${system_label}" ]; then
-    set label=${system_label}
-  else
-    set recoverylabel=${recovery_label}
-  fi
+  # label is kept around for backward compatibility
+  set label=${system_label}
```

Comment:
I do not entirely remember why label and recoverylabel were required in terms of backward compatibility. However I suspect that if label is still required recoverylabel is also. Need to look at it closer to asses it.